includes/admin/settings/display-settings.php

```diff
@@ -90,6 +90,7 @@ function edd_admin_header() {
 	}
 
 	?>
+	<?php do_action( 'edd-admin-top-header' ); ?>
```

Comment:
If this new hook is not being used, does it need to be introduced at this point?

If it does, WPCS wants it to use underscores, not hyphens.

Reply:
@robincornett we can remove it, I was playing around with a few ideas, but ultimately I found a way that works better.